### ISSUE TITLE
2D polygon graphics

### ISSUE THREAD

**Issue by kushalkolar** [fastplotlib/fastplotlib#487]
2D Polygons meshes

**Comment by kushalkolar** [comment]
for manipulating meshes: https://github.com/pygfx/gfxmorph

**Comment by kushalkolar** [comment]
@almarklein this can just be done with this right:

https://github.com/fastplotlib/fastplotlib/blob/main/fastplotlib/utils/triangulation.py

**Comment by almarklein** [comment]
Yeah, the polygon selector tool basically draws ... a polygon, so similar logic can be used for a polygon graphic! 👍
